.vscode/settings.json

```diff
@@ -81,5 +81,6 @@
   "terminal.integrated.scrollback": 10000,
   "workbench.editor.enablePreview": false,
   "yaml.format.enable": true,
-  "yaml.maxItemsComputed": 1000000
+  "yaml.maxItemsComputed": 1000000,
+  "swift.debugger.debugAdapter": "CodeLLDB"
```

Comment:
The new LLDB-DAP that is in the toolchains is a bit buggy, so set this to use the good ol' CodeLLDB for now.